modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryStorageEngine.java

```diff
@@ -353,6 +355,15 @@ public void destroyMvTable(int tableId) {
         }
     }
 
+    @Override
+    public void addTableMetrics(StorageTableDescriptor tableDescriptor, Consumer<Metric> metricConsumer) {
+        PersistentPageMemoryDataRegion region = regions.get(tableDescriptor.getStorageProfile());
+
+        assert region != null : "Adding metrics to the table with non-existent data region: " + tableDescriptor;
```

Comment:
IMHO, it would be nice to use only one approach to prepare a message that is used for logs, asserts, etc. https://cwiki.apache.org/confluence/display/IGNITE/Java+Logging+Guidelines#JavaLoggingGuidelines-MessageFormat
```
Adding metrics to the table with non-existent data region [tableDescriptor=" + tablecDescriptor + ']';
```
Anyway, it's up to you.